libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -284,14 +284,6 @@ static DNNReturnType get_input_ov(void *model, DNNData *input, const char *input
                 return DNN_ERROR;
             }
 
-            // The order of dims in the openvino is fixed and it is always NCHW for 4-D data.
-            // while we pass NHWC data from FFmpeg to openvino
-            status = ie_network_set_input_layout(ov_model->network, input_name, NHWC);
-            if (status != OK) {
-                av_log(ctx, AV_LOG_ERROR, "Input \"%s\" does not match layout NHWC\n", input_name);
-                return DNN_ERROR;
-            }
-
             input->channels = dims.dims[1];
```

Comment:
merge this patch together with the second patch, otherwise, we need to explain why here

Reply:
OK, sure